ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/AuthorizationFunctionalTest.java

```diff
@@ -122,39 +130,31 @@ public void shouldDenyAccess() {
     // Then:
     expectedException.expect(AssertionError.class);
     expectedException.expectMessage(
-        String.format("Access denied to User:%s", VALID_USER1.username)
+        String.format("Access denied to User:%s", USER1.username)
     );
 
     // When:
-    makeKsqlRequest(VALID_USER1, "SHOW TOPICS;");
+    makeKsqlRequest(USER1, "SHOW TOPICS;");
   }
 
   @Test
   public void shouldAllowAccess() {
     // Given:
-    givenAuthorizedUser(VALID_USER1);
+    givenAuthorizedUser(USER1);
 
     // When:
-    final List<KsqlEntity> results = makeKsqlRequest(VALID_USER1, "SHOW TOPICS;");
+    final List<KsqlEntity> results = makeKsqlRequest(USER1, "SHOW TOPICS;");
 
     // Then:
-    assertSuccessful(results);
+    final List<KafkaTopicInfo> topics = ((KafkaTopicsList)results.get(0)).getTopics();
```

Comment:
To be extra thorough we could also assert that `results.size()` is 1, but it's not a huge deal.